muikku/src/main/webapp/index.xhtml

```diff
@@ -212,7 +212,9 @@
 		    <div class="lg-flex-cell12-4 md-flex-cell-8 sm-flex-cell-full lg-flex-cell-first md-flex-order-3 sm-flex-order-3">
 		      <div class="box box-max-width">
 		        <div id="visits"></div> 
-		        <div id="workspaces"></div>
+		        <ui:fragment rendered="#{sessionBackingBean.isStudent}">
```

Comment:
Changes needed. If staff member has workspaces, the section should render normally. If not, it should not be shown. For students without workspaces, it should render with an instructional message about signing up.